src/js/eventHandler.spec.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { callbackOnFormSubmit, callbackWhenEnterIsPressed } from './eventHandlers';
+
+document.body.innerHTML = `
+<form class="chat-window__sender-features">
+  <label for="message-area" class="visually-hidden">Message area</label>
+  <textarea
+    name="message-area"
+    id="message-area"
+    class="chat-window__input">Hello World</textarea>
+  <button class="chat-window__send-button" type="submit">Send</button>
+</form>
+`;
+
+describe('eventHandler.js', () => {
```

Comment:
```suggestion
describe('#callbackWhenEnterIsPressed', () => {
```

The entire file should be considered to be the tests for `eventHandler.js`, this shows that this block of tests apply to a specific function. I would wrap the test(s) for the other function in a similar `describe` block.